core/cap-0035.md

```diff
@@ -71,8 +71,8 @@ trustline is created to authorize a `ClawbackOp` operation submitted by the
 issuer account.
 
 A claimable balance inherits its clawback enabled status from the account
-creating it and a `ClawbackClaimableBalanceOp` operation is only valid for
-claimable balances created from accounts that have clawback enabled.
+creating the asset and a `ClawbackClaimableBalanceOp` operation is only valid for
+claimable balances created for assets whose issuers have clawback enabled.

```

Comment:
Shouldn't it be `account creating the balance entry` instead of `account creating the asset`? 

Also, I feel like the last line implies the dependency is on the issuer account flag, and not the trustline flag, which is not the case. Maybe another way to write it would be `...is only valid for
claimable balances created using a trustline with clawback enabled.`

Reply:
Agreed, the claimable balance inherits the state from the trustline on the account, not the account creating the asset.

Reply:
@bartekn Can you make these changes?
- `the asset` to `the claimable balance,`
- `for assets whose issuers have clawback enabled` to `by accounts whose trustline for the asset has clawback enabled`

Reply:
4fcd112